### PR TITLE
Added incidence-trend tool to showcase list

### PR DESCRIPTION
**Pull request by lud-hu**

### CODE REVIEW

**Review by Rubber1Duck — CHANGES_REQUESTED** [review]
in docs/index.md this should be `7-Tage-Inzidenz Verlauf mit Vergleich beliebiger Städte/Länder [https://incidence-trend.web.app/](https://incidence-trend.web.app/)` so the link is clickable in the running server!

File: docs/index.md [hunk 1]
```diff
@@ -124,6 +124,7 @@ Now you can access the server at `http://localhost:8080`.
 - COVID-19 Impfdauer-Rechner (https://www.impfdauer.de)
 - Discord Corona Bot (https://github.com/markxoe/covid-bot)
 - COVID-19 Dashboard (https://covid.beyerleinf.de)
+- 7-Tage-Inzidenz Verlauf mit Vergleich beliebiger Städte/Länder (https://incidence-trend.web.app/)
```
Comment (Rubber1Duck) [inline docs/index.md:128]: this should be 
`7-Tage-Inzidenz Verlauf mit Vergleich beliebiger Städte/Länder [https://incidence-trend.web.app/](https://incidence-trend.web.app/)`

**Review by Rubber1Duck — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (lud-hu) [reply docs/index.md:128]: Thanks! You're totally right! 👍 I'll add it to the PR.

**Review by Rubber1Duck — APPROVED** [review]

**Comment by Rubber1Duck** [comment]
PR #166 is a replacement for this PR! this one can be closed

**Comment by Rubber1Duck** [comment]
@lud-hu please close this PR! Your Data is implemented by PR #166 !

lud-hu

**Pull request closed without merging**